templates/base/Dockerfile

```diff
@@ -3,10 +3,35 @@ FROM --platform=${TARGETPLATFORM} ubuntu:jammy AS ubuntu-jammy
 COPY resources/etc/created-date /etc/created-date
 
 RUN apt-get update && \
-	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes bash ca-certificates curl jq less libnss3 telnet tini tree unzip && \
+	DEBIAN_FRONTEND=noninteractive TZ=Etc/UTC apt-get install --no-install-recommends --yes build-essential ca-certificates curl jq less libnghttp2-dev libnss3 libssl-dev nghttp2 telnet tini tree unzip && \
```

Comment:
@Wykenz, why do we need the following packages?

- build-essential
- libnghttp2-dev
- libssl-dev
- nghttp2

If some of them are necessary just for the `curl` install then we can remove them after the `curl` install